bin/upgrade.bash

```diff
@@ -6,6 +6,7 @@ ROOT="$(readlink -f "${here}/../")"
 
 CK8S_STACK="$(basename "$0")"
 export CK8S_STACK
+export CLUSTER="${1}"
```

Comment:
Similar to [this discussion](https://github.com/elastisys/compliantkubernetes-apps/pull/1778#discussion_r1362180161) in ck8s-apps you could choose to use this as either an argument or exported variable but preferably not both. I suggest using the same solution for both repos, whatever it ends up being.

Reply:
The most recent commit should reflect how it was done in ck8s-apps, please have another look